x/auth/vesting/msg_server.go

```diff
@@ -163,11 +161,14 @@ func (s msgServer) CreatePeriodicVestingAccount(goCtx context.Context, msg *type
 	}
 
 	var totalCoins sdk.Coins
-
 	for _, period := range msg.VestingPeriods {
 		totalCoins = totalCoins.Add(period.Amount...)
 	}
 
+	if err := bk.IsSendEnabledCoins(ctx, totalCoins...); err != nil {
+		return nil, err
+	}
+
```

Comment:
should we update changelog?

Reply:
Yes, but I'll do that in the backport PR, just to have on conflict less to fix in v0.47 :)